.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "python.testing.pytestArgs": [
+    "gherkin/python/gherkin/test"
+  ],
+  "python.testing.unittestEnabled": false,
+  "python.testing.pytestEnabled": true
+}
```

Comment:
I think this file should go into `.gitignore`.

Reply:
This is IDE specific: provides a development container co-located with your repository:

https://code.visualstudio.com/docs/devcontainers/containers

It could provide an easier experience for contributors to get started as well as standardise your development environment.

I created these before I found the [cucumber/build](https://github.com/cucumber/build) repository that I could reference.

I'll remove.